tcmalloc/BUILD

```diff
@@ -18,14 +18,28 @@
 # https://github.com/google/tcmalloc/tree/master/docs/design.md for a high-level description of
 # how this malloc works.
 
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
 load("//tcmalloc:copts.bzl", "TCMALLOC_DEFAULT_COPTS")
 load("//tcmalloc:variants.bzl", "create_tcmalloc_benchmark", "create_tcmalloc_libraries", "create_tcmalloc_testsuite")
 
 package(default_visibility = ["//visibility:private"])
 
 licenses(["notice"])
 
-exports_files(["LICENSE"])
+exports_files(["LICENSE", "tcmalloc.h"])
+
+bool_flag(
+    name = "alwayslink",
+    build_setting_default = True,
+)
+
+config_setting(
+    name = "unlinked",
+    flag_values = {
+        ":alwayslink": "false",
+    },
+    visibility = ["//visibility:public"],
+)
```

Comment:
`alwayslink` generates a `.lo` file which is not recognized by `rustc`. This flags allws users to disable it for rust specifically.